ethstorage/p2p/discovery.go

```diff
@@ -327,11 +371,13 @@ func (n *NodeP2P) DiscoveryProcess(ctx context.Context, log log.Logger, l1ChainI
 			log.Info("Stopped peer discovery")
 			return // no ctx error, expected close
 		case found := <-randomNodesCh:
+			// get the most recent version of the node record in case it change deal to the remote node TCP or UDP port change.
+			node := n.dv5Udp.Resolve(found)
```

Comment:
As a node with a public IP, how can we verify that a node running in a docker can be accessed through the correct ENR?

Reply:
we can add API to query the peer ENR or out the ENR to metrics later.